src/Infrastructure.Dapper/Repositories/UserRepository.cs

```diff
@@ -220,30 +220,13 @@ public async Task UpdateUserKeyAndEncryptedDataAsync(
         try
         {
             // Update user
-            await using (var cmd = new SqlCommand("[dbo].[User_UpdateKeys]", connection, transaction))
```

Comment:
❓ : This looks like a good change to me, but there is a small bit of functionality difference where `User_UpdateKeys` used to make `AccountRevisionDate` default to `RevisionDate` if null. 

If that's not an issue, can you investigate what our proper stored proc deprecation processes are as I don't believe `User_UpdateKeys` is used anywhere else now?

Reply:
Interesting, on the entityframework repository this does not have the same fallback behavior.

(Also, my changes just ignored the entity framework, so luckily caught that here).